FtpProvider/FtpProvider/FtpTypeProvider.fsproj

```diff
@@ -51,7 +51,6 @@
   <ItemGroup>
     <Compile Include="ProvidedTypes.fsi" />
     <Compile Include="ProvidedTypes.fs" />
-    <Content Include="packages.config" />

```

Comment:
"What's this TODO for?"

when comments get in the way of just reading the code, I came up with an
annotation convention.  i.e. TODO.6 refers a specific TODO item below.  In
my minds eye I imagined a tooling extension in VS for this.  Even in a raw
form I've found this useful though.  Since just a personal style, I'll
remove them.

On 10 September 2015 at 05:50, Steven Taylor <taylste@gmail.com> wrote:

>
> In FtpProvider/FtpProvider/FtpTypeProvider.fsproj
>> <https://github.com/dsyme/FtpTypeProviderExample/pull/5#discussion_r38634948>
>> :
>>
>> > @@ -51,7 +51,6 @@
>> >    <ItemGroup>
>> >      <Compile Include="ProvidedTypes.fsi" />
>> >      <Compile Include="ProvidedTypes.fs" />
>> > -    <Content Include="packages.config"
>>
>> Why is this removed?
>>
>> Not removed.  I merged the two <ItemGroup> nodes for .dlls and source
> code into one.
>
> The VS IDE stopped working correctly with the click-reference expand
> behaviour.  I initially thought that the cause might be a HD corruption or
> a  change in the .fsproj file.  I was looking for the cause + did a little
> editing while there.   The cause was actually a VS plug-in -- probably the
> GitHub one, because the most recent update fixed the issue.  I'll put it
> back to the way it was.
>